telegram/callbackquery.py

```diff
@@ -196,3 +196,27 @@ def edit_message_reply_markup(self, reply_markup, *args, **kwargs):
                                                       chat_id=self.message.chat_id,
                                                       message_id=self.message.message_id,
                                                       *args, **kwargs)
+
```

Comment:
flake8 has some issues with this, 
![image](https://user-images.githubusercontent.com/3002987/56914286-0316e380-6ab4-11e9-8898-2a6cd0ec2ff8.png)